### PR TITLE
Add version edit for Local Git deployment to use HEAD

### PR DESCRIPTION
**Pull request by ttomsu**

### CODE REVIEW

File: setup/install/environment.md [hunk 1]
```diff
@@ -123,6 +123,8 @@ the following command.
 
 ```
 hal config deploy edit --type localgit --git-origin-user=<YOUR_GITHUB_USERNAME>
+
+hal config version edit --version branch:upstream/master
```
Comment (duftler) [inline setup/install/environment.md:127]: The description just above ends with "...command" but there are two commands.

Seems like this 2nd command should have its own code block with its own single sentence description of why one might want to also issue it after setting the type to `localgit`.

**Review by duftler — APPROVED** [review]
LGTM after addressing comment.

**Review by dorbin — APPROVED** [review]

File: setup/install/environment.md (hunk 1 above)
Comment (ttomsu) [reply setup/install/environment.md:127]: Expanded the text a bit.

**Review by duftler — APPROVED** [review]
LGTM.

**Pull request merged**